pom.xml

```diff
@@ -657,6 +657,116 @@
             </build>
         </profile>
 
+        <profile>
+            <id>payara-micro-remote</id>
+
+            <properties>
+                <!-- default port -->
+                <payara.httpPort>8080</payara.httpPort>
+                <!-- JMS and JAXWS not supported by Micro -->
+                <skipJMS>true</skipJMS>
+                <skipJAXWS>true</skipJAXWS>
+
+                <!-- Being a WebProfile++, Micro does not support ear archives -->
+                <skipEAR>true</skipEAR>
+
+                <!-- Client-cert needs complicated cert setup, which hasn't been done -->
+                <skipServletClientCertificate>true</skipServletClientCertificate>
+            </properties>
+
+            <dependencies>
+
+                <!-- Java EE based client dependencies to contact a server via WebSocket or REST -->
+                <!-- Not needed by the container, but some tests seem to expect JAXRS client implementation to
+                be on the classpath -->
+                <dependency>
+                    <groupId>fish.payara.arquillian</groupId>
+                    <artifactId>payara-client-ee7</artifactId>
+                </dependency>
+
+                <!-- The actual Arquillian connector -->
+                <dependency>
+                    <groupId>fish.payara.arquillian</groupId>
+                    <artifactId>payara-micro-remote</artifactId>
+                    <version>1.0.Beta4-SNAPSHOT</version>
```

Comment:
should we be using a snapshot?

Reply:
No we shouldn't, but this serves as a test case that payara/ecosystem-arquillian-connectors#1 works.

I'll file an issue for releasing 1.1 and then would update it.

Reply:
Properties for versions are nice :) Everything else uses them

Reply:
> Properties for versions are nice :) Everything else uses them

I agree and was about to change it. However profile-dependent dependencies in javaee samples projects seem to be an exception. With `payara-client-ee7` being exception of exception :)

I believe this makes sense from upstream project's point of view -- vendor dependencies should not be in top level dependency management, rather in respective profiles. And making profile-specific dependency management section doesn't bring anything.

But it is possible that every `payara-*` profile would include `payara.arquillian.version` and used it for its ecosystem dependency. Would that be a viable option?